pkgs/applications/science/math/R/default.nix

```diff
@@ -103,6 +112,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   passthru.tests.pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
 
+  # make tex output available to texlive.combine
+  passthru.tlType = "run";
```

Comment:
I am not so sure but isn't `tlDeps` needed?

Reply:
Good point, I have added the packages used in `Sweave.sty` and similar as dependencies.